.travis.yml

```diff
@@ -1,15 +1,35 @@
 language: java
-jdk: oraclejdk8
-branches: 
-only:
-- master
+jdk:
+- oraclejdk8
 cache:
   directories:
   - "$HOME/.m2"
-after_success:
-- openssl aes-256-cbc -pass pass:$ENCRYPTION_PASSWORD -in $GPG_DIR/pubring.gpg.enc -out $GPG_DIR/pubring.gpg -d
-- openssl aes-256-cbc -pass pass:$ENCRYPTION_PASSWORD -in $GPG_DIR/secring.gpg.enc -out $GPG_DIR/secring.gpg -d
-- "$GPG_DIR/publish.sh"
+install:
+#- sudo apt-get install -y gnupg2
```

Comment:
we need this. I have added it. I only commented certain parts to reduce the build time.